research/src/components/concepts.css

```diff
@@ -0,0 +1,26 @@
+.concepts {
+    display: flex;
+    justify-content: space-between;
+}
+
+label[for=descriptions] {
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+}
+
+#descriptions {
```

Comment:
can we guarantee that only one of these will be present on a page at the time, is it worth converting to a class instead?

Reply:
From what I see this is top-level component which represent single research page. I do not see how it can be duplicated.